database/migrations/2024_03_23_073129_create_shop_orders_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('shop_orders', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('customer_id');
+            $table->unsignedBigInteger('address_id');
+            $table->unsignedBigInteger('shipping_method_id');
+            $table->unsignedBigInteger('status_id');
+            $table->unsignedBigInteger('order_total');
```

Comment:
is this the total amount or the total quantity ? if amount, please use double

Reply:
Yes it is the total price amount . We will use double.